src/main/resources/templates/index.html

```
@@ -20,6 +20,10 @@
 <nav th:replace="~{fragment/navbar :: copy}"></nav>
 
 
+<!-- 사이드바 fragment 사용 -->
+<aside th:replace="~{fragment/sidebar :: copy}"></aside>
+
+
 <!-- 네비게이션 바의 높이만큼 상단 여백 추가 -->
 <div class="pt-16">
```

HTMLHint (1.5.0)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div class="pt-16"> ] on line 28.

(tag-pair)